application/account.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+class Account
+  attr_reader :name, :login, :password, :errors, :cards
+
+  include Database
+
+  VALID_RANGE = {
+    age: (23..89),
+    login: (4..20),
+    password: (6..30)
+  }.freeze
+
+  def initialize(data)
+    @name = data[:name]
+    @age = data[:age].to_i
+    @login = data[:login]
+    @password = data[:password]
+    @cards = []
+    @errors = []
+  end
+
+  def valid?
+    validation
```

Comment:
it's better to name a method as a verb: `validate`

Reply:
named method validate